examples/for-tests/package.json

```diff
@@ -4,8 +4,8 @@
     "private": true,
     "dependencies": {
         "axios": "^0.21.0",
```

Comment:
Are we testing with react 17 and with react 18?

Reply:
Not yet, I'll add that in a separate PR. (Most likely next week)

Reply:
okay.